pandas/core/window.py

```diff
@@ -879,13 +879,64 @@ def f(arg, *args, **kwargs):
                            ddof=ddof, **kwargs)
 
     _shared_docs['var'] = dedent("""
-    %(name)s variance
+    Calculate unbiased %(name)s variance.
+
+    Normalized by N-1 by default. This can be changed using the `ddof`
+    argument.
 
     Parameters
     ----------
     ddof : int, default 1
         Delta Degrees of Freedom.  The divisor used in calculations
-        is ``N - ddof``, where ``N`` represents the number of elements.""")
+        is ``N - ddof``, where ``N`` represents the number of elements.
+    *args
+        Under Review.
```

Comment:
@jorisvandenbossche ?

Reply:
It was mentioned that the exact content for what should be documented for args/kwargs was to be determined. If you'd rather me add the documentation I can do that too.

Reply:
I think we're noting that they're accepted, but not used.